src/emcpy/plots/domain.py

```diff
@@ -0,0 +1,108 @@
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+import matplotlib.pyplot as plt
+
+__all__ = ['get_domain']
+
+
+def _global(fig):
+    """
```

Comment:
question:
Just because this is a private method, should we not document it like we do public methods?

Reply:
I think some documentation is important (like what he has here) but do we need to do the whole args: returns: thing if it is private? Pdoc doesn't even render them, right?

Reply:
Doesn't hurt to document it. I wasn't sure how documenting private methods worked. I will add them to `__all__` for this script and `map2d.py`.

Reply:
I'm not sure we want to add them to `_all_`, because they are private. Are there cases it would be used by the average user directly?

Reply:
I doubt a user will ever need to call these.  It is for future proofing ourselves, so we (developers) know what the in and out are.

Reply:
It should not be in `__all__`.  `__all__` contains the user API.

Reply:
ok so add the docstrings but keep them out of the __all__, that way we see what's up if we are editing the code but it doesn't clutter up the documentation?

Reply:
yeah, that was my thought